src/containers/BooksForm.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { createBook } from '../actions/index';
+
+class BookForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      title: '',
+      category: '',
+    };
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleChange(e) {
+    this.setState({
+      title: e.target.title,
+      category: e.target.category,
+    });
+  }
+
+  handleSubmit(e) {
+    e.preventDefault();
+    const { title, category } = this.state;
+    const book = {
+      id: Math.floor(Math.random() * 100),
```

Comment:
* [ ]  This line of code does not guarantee unique id creation, random numbers are not necessarily unique, here are robust way to get unique ids, kindly use packages like `shortId` or `uuid` instead.